consensus/objs/accusations.go

```diff
@@ -0,0 +1,68 @@
+package objs
+
+import (
+	"errors"
+
+	"github.com/google/uuid"
+)
+
+type AccusationState uint8
+
+const (
+	// Created means the accusation has been identified but not yet persisted in any way.
+	Created AccusationState = 0
```

Comment:
Might want to use [iota](https://github.com/golang/go/wiki/Iota) for enum-like consts. Also, not a bad idea to have an `Unset/Invalid AccusationState` for the 0 value otherwise uninitialized values will be considered `Created`

Reply:
✅